packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -58,18 +63,28 @@ export class InteractionTracker {
         console.log('Click detected');
         if (this.currentEventTracingZone === undefined) {
           const traceId = randomTraceId();
-          this.currentEventTracingZone = Zone.root.fork({
-            name: traceId,
-            properties: {
-              isTracingZone: true,
+          const spanOptions = {
```

Comment:
the Zone.prototype.runtask() function is getting pretty long. If you see a way to cleanly extract a helper function that help keep it clear.